tests/e2e/tests_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package integration_tests_test
```

Comment:
The package names are slightly confusing here. Is the `integration_tests_test` testing the integration tests or just running the tests?